kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -114,7 +114,9 @@ grCollectionsPanel.controller('GrNodeCtrl',
     };
 
     subscribe$($scope, pathWithImages$, ({path, images}) => {
-       collections.addImagesToCollection(images, path).then(() => ctrl.saving = false);
+       collections.addImagesToCollection(images, path).then(() => {
+           $scope.$apply(() => ctrl.saving = false);
```

Comment:
And if we're in the subscribe, potentially out of angular's digest scope.
Hypothesis, but hard to test.